### DISCUSSION TITLE
GINAN estimation the ionosphere

### DISCUSSION THREAD

**Discussion by chenst1227** [GeoscienceAustralia/ginan#77] [Q&A]
Dear Ginan team
I would like to do some ionospheric solving and estimation using the GINAN software. In the latest version of the software I did not find a configuration file on how to solve the ionosphere using the GINAN executable program. What should I do please? I would appreciate it if you could explain this issue to me. Thank you very much, and I look forward to your reply.

**Comment by ronaldmaj** [comment]
Hi [chenst1227](https://github.com/chenst1227),

Thanks for your question, we recently moved to version 3 of the Ginan software and are still working through updating our various example configuration files - the ionospheric processing being one of them.
Unfortunately, that one is not ready for public release yet, but we are currently working on it. Once we have that ready, that should be available on our `develop-weekly` branch. I will let you know when that has occurred.
Please don't hesitate to ask if you have any other questions or queries :)

**Comment by chenst1227** [comment]
Hi,


Thank you for your prompt responses. I will continue to monitor your "develop-weekly" branch closely!




---Original---
From: "Ronald ***@***.***&gt;
Date: Fri, Mar 22, 2024 12:38 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [GeoscienceAustralia/ginan] GINAN estimation the ionosphere(Discussion #77)




 
Hi chenst1227,
 
Thanks for your question, we recently moved to version 3 of the Ginan software and are still working through updating our various example configuration files - the ionospheric processing being one of them.
 Unfortunately, that one is not ready for public release yet, but we are currently working on it. Once we have that ready, that should be available on our develop-weekly branch. I will let you know when that has occurred.
 Please don't hesitate to ask if you have any other questions or queries :)
